Adding information on getting a Serpent License (#194)
arfc/arfc.github.io: manual/guides/getting_serpent_license.md

---
layout: manual
title: Acquiring a Serpent License
subtitle: "Step-by-step instructions for getting a Serpent license"
permalink: /manual/guides/getting_serpent_license
---

# Register at RSICC

- Go to [rsicc.ornl.gov](https://rsicc.ornl.gov/)
- Click on "Registration"
- Fill out the registration form. Once this is done,
you should receive an RSICC Pass Number. You will
need this for the next step.

# Request the Software at RSICC

- Go to [rsicc.ornl.gov](https://rsicc.ornl.gov/)
- Go to "Requests"
- Log in and fill out the request form.

# The Request Form

The request form is where you state your reason
for needing Serpent. You are going to need the
sponsoring professor(s), the professor(s) University
email, your University email, and the course
name(s)/number(s) with their description
(or your project description if you are doing
your own research).

For the project description, there is a general template
below that has been used by people that obtained their
license:

```markdown

> I am a graduate student in the Advanced Reactors and Fuel Cycles group at the University

Reviewer: ```suggestion
I am a graduate student in the Advanced Reactors and Fuel Cycles group at the University of Illinois in Urbana-Champaign, which is led by Prof. Kathryn Huff.
```

Reviewer: I would put it in the same line, otherwise, it looks different depending on the window size.

> of Illinois in Urbana-Champaign, which is led by Prof. Kathryn Huff.

Reviewer: ```suggestion

```

My research is funded by _______. My use of Serpent will contribute to _______.
I intend to use Serpent to _______. In the near term, this work will _______.

> This work will contribute to my thesis work in _______. Data and software management

Reviewer: ```suggestion
This work will contribute to my thesis work in _______. 
Data and software management in this work will abide by all license guidelines and will be informed by the export control officer at the University of Illinois.
```

> in this work will abide by all license guidelines and will be informed

Reviewer: ```suggestion
```

> by the export control officer at the University of Illinois.

Reviewer: ```suggestion
```

```

Here is an example where the blanks have been filled in:

```markdown

> I am a graduate student in the Advanced Reactors and Fuel Cycles group at the University

Reviewer: ```suggestion
I am a graduate student in the Advanced Reactors and Fuel Cycles group at the University of Illinois in Urbana-Champaign, which is led by Prof. Kathryn Huff.
```

> of Illinois in Urbana-Champaign, which is led by Prof. Kathryn Huff.

Reviewer: ```suggestion
```

My research is funded by a graduate fellowship from the U.S. Nuclear Regulatory Commission.

> My use of Serpent will contribute to the licensing and regulatory mission of the U.S. NRC

Reviewer: ```suggestion
My use of Serpent will contribute to the licensing and regulatory mission of the U.S. NRC through the investigation of advanced nuclear reactor safety.
```

> through investigation of advanced nuclear reactor safety. I intend to use Serpent

Reviewer: ```suggestion
I intend to use Serpent to run neutronics simulations of advanced reactor types relevant to the future of the US nuclear energy system.
```

> to run neutronics simulations of advanced reactor types relevant to the future

Reviewer: ```suggestion
```

> of the US nuclear energy system. In the near term, this work will emphasize generation

Reviewer: ```suggestion
In the near term, this work will emphasize generation of group constants supporting safety analysis of Molten Salt Reactor core designs.
```

> of group constants supporting safety analysis of Molten Salt Reactor core designs.

Reviewer: ```suggestion
```

> This work will contribute to my thesis work in advanced reactor multi-physics methods

Reviewer: ```suggestion
This work will contribute to my thesis work in advanced reactor multi-physics methods for safety analysis and design optimization.
```

> for safety analysis and design optimization. Data and software management in this work

Reviewer: ```suggestion
Data and software management in this work will abide by all license guidelines and will be informed by the export control officer at the University of Illinois.
```

> will abide by all license guidelines and will be informed by the export control officer

Reviewer: ```suggestion
```

> at the University of Illinois.

Reviewer: ```suggestion

```
